spark-connector/v3.4/spark/build.gradle.kts

```diff
@@ -135,6 +135,18 @@ tasks.test {
   }
 }
 
```

Comment:
could you use a variable to represent `spark-${sparkMajorVersion}_${scalaVersion}`?

Reply:
OK